appveyor/Convert-FileUrisToGitHubUris.ps1

```diff
@@ -134,5 +134,5 @@ for ($i = 0; $i -lt $sarifLog.runs.Count; $i++){
 
 $newExtension = ".github.sarif"
 $writeLocation = [System.IO.Path]::ChangeExtension($logFilePath, $newExtension)
-$writeFile = $sarifLog | ConvertTo-Json -depth 999
```

Comment:
Kind of. They changed ConvertTo-Json to throw if the value is set to anything larger than 100.